Dockerfile

```diff
@@ -56,19 +56,19 @@ RUN apt-get install -y \
     libssl-dev \
     libsasl2-dev \
     libbson-dev \
-    git \
-    curl
```

Comment:
we use curl so I think it should stay

Reply:
yes, but it is still in dockerfile. This one was second(they were doubled) one, so I removed it